test/README.md

```diff
@@ -97,3 +97,22 @@ From here you can interact via the websockets exposed in the direct links, in th
 - Alice (relay): `35005`
 - Bob (relay): `37613`
 - Collator (storage-hub): `45615`
+
+### Generating new Type Interfaces
+
+This repo uses Parity's [polkadot-api](https://github.com/polkadot-api/polkadot-api) AKA PAPI.
+To generate new type interfaces run the following in `/test`:
+
+```sh
+bun zombie:run:native
+```
+
+In another terminal window in `/test`:
+
+```sh
+bun scalegen
```

Comment:
I tried running this, works as expected, although it generates modifications on the `.scale` files. Is that expected? I would have guess that they would already be up to date, and me running it wouldn't make changes.

Reply:
scalegen's output depends on what parachain and relaychain is being run against. I'm running against the SH in this checkout + relaychain 1.9